Magento_Catalog/templates/product/list.phtml

```diff
@@ -131,7 +131,7 @@ use Magento\Framework\App\Action\Action;
                                     aria-label="<?= $block->escapeHtmlAttr(__('Product description list')) ?>"
                                     tabindex="0"
                                 >
-                                    <?= $_product['selling_features_bullets'] ?>
+                                    <?= $block->escapeHtml($_product['selling_features_bullets']) ?>
```

Comment:
here I think we should use noEscape as here we will have html to show

Reply:
I created issue in module: https://github.com/SnowdogApps/magento2-bullet-points/issues/1